public/index.html

```diff
@@ -18,7 +18,7 @@
 -->
 
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Will we have to refresh this as the shopper selects different languages?

Reply:
In index.tsx file value for the lang attribute is set to the selected language.

Reply:
Thanks @yulia-dnistrian, I see that change now. This is great!